server/irmaserver/api.go

```diff
@@ -265,14 +305,14 @@ func (s *Server) SubscribeServerSentEvents(w http.ResponseWriter, r *http.Reques
 
 	var session *session
 	if requestor {
-		session = s.sessions.get(token)
+		session, _ = s.sessions.get(token) // SSE can only be used with storeType memory which does not contain errors.
 	} else {
-		session = s.sessions.clientGet(token)
+		session, _ = s.sessions.clientGet(token) // SSE can only be used with storeType memory which does not contain errors.
```

Comment:
How do you mean that? That the irma server gives a warning and performs sse anyway (knowing that it might miss certain cases)? Or just that we keep this open until we know whether it is possible or not?

Reply:
Both:
* For now we emit a warning and allow it, for consistency with the code that validates the configuration;
* When later we are certain that Redis and SSE can or cannot be combined, we remove the warning or make it into an error, respectively.

Reply:
For the minimal version, we will not enable SSE (decided by team). Therefore the combination must cause an error. A single warning would let the server start up and would result in the client thinking it can use SSE but never receiving updates.